hedera-node/src/main/java/com/hedera/services/contracts/operation/HederaOperationUtil.java

```diff
@@ -201,4 +206,26 @@ public static long newContractExpiryIn(MessageFrame frame) {
 
 		return supplierExecution.get();
 	}
+
+	// TODO: Add JavaDocs
+	public static void setOriginalReadValue(
```

Comment:
naming nit, this makes it feel like a java property.  Perhaps `cacheExistingValue` or `recordOriginalForStateDiff`?  Such a name also communicates in the places it's being called why we are doing it.